pandas/__init__.py

```diff
@@ -4,17 +4,13 @@
 __docformat__ = 'restructuredtext'
 
 try:
-    from . import hashtable, tslib, lib
-except Exception:  # pragma: no cover
-    import sys
-    e = sys.exc_info()[1]  # Py25 and Py3 current exception syntax conflict
-    print(e)
-    if 'No module named lib' in str(e):
-        raise ImportError('C extensions not built: if you installed already '
-                          'verify that you are not importing from the source '
-                          'directory')
-    else:
-        raise
+    from pandas import hashtable, tslib, lib
+except ImportError as e:  # pragma: no cover
+    module = str(e).lstrip('cannot import name ')  # hack but overkill to use re
```

Comment:
hmm, is it necessary to print the exception? I assume the interpreter will just quit in any event, not sure why was done that way previously. any thoughts?